quickstarts/basic-dashboard/config.yml

```diff
@@ -0,0 +1,28 @@
+id: e4abebd7-cce0-4474-b070-e2b0007f0e75
```

Comment:
This will be auto generated after merging the pull request, please remove this field for now.
```suggestion
```